src/components/Navbar/SettingsDrawer/SettingsDrawer.module.scss

```diff
@@ -0,0 +1,59 @@
+@use "src/styles/breakpoints";
+@use "src/styles/constants";
+
+.container {
+  background: var(--color-background-default);
+  position: fixed;
+  height: 100vh;
+  width: 100%;
+  top: 0;
+  bottom: 0;
+  z-index: var(--z-index-header);
+  transition: var(--transition-regular);
+  overflow-y: auto;
+  overflow-x: hidden;
+  overscroll-behavior-y: contain;
+  right: -100%;
+
+  @include breakpoints.tablet {
+    width: constants.$side-menu-desktop-width;
+    right: calc(0 - constants.$side-menu-desktop-width);
+  }
+}
+
+.containerOpen {
+  box-shadow: var(--shadow-normal);
+  right: 0;
+  @include breakpoints.tablet {
+    right: 0;
+  }
```

Comment:
do we need this?

Reply:
remove this and create drawer component, seperate PR